src/rat/run_rat.py

```diff
@@ -93,8 +93,14 @@ def run_rat(config_fn, operational_latency=None):
             # Run RAT forecast for basin if forecast is True
             if config['PLUGINS']['forecast']:
                 log.info('############## Starting RAT forecast for '+config['BASIN']['basin_name']+' #################')
-                no_errors = forecast(config, log)
+                forecast_no_errors = forecast(config, log)
                 log.info('############## RAT forecast finished for '+config['BASIN']['basin_name']+ ' with '+str(no_errors)+' error(s). #################')
+                if(forecast_no_errors>0):
+                    log.info('############## RAT run finished for '+config_copy['BASIN']['basin_name']+ ' with '+str(forecast_no_errors)+' error(s). #################')
+                elif(forecast_no_errors==0):
+                    log.info('############## Succesfully run RAT for '+config_copy['BASIN']['basin_name']+' #################')
+                else:
+                    log.error('############## RAT run failed for '+config_copy['BASIN']['basin_name']+' #################')
```

Comment:
In place of line 97, I think lines 98 to 103 will suffice. Only thing is it will be good to say 'RAT forecasting run finished' or 'Successfully run RAT forecasting plugin' just to make it clear that these status are for forecasting as it is using forecast_no_errors. Same thing below for multiple basins.